static-site/docs.udon

```diff
@@ -1,13 +1,55 @@
-        title/'Technical overview'
-    ==
-;>
+1 :-  :~  navhome/'/docs/'
+2         navuptwo/'true'
+3         next/'true'
+4         sort/'1'
+5         title/'Introduction'
+6     ==
+7 ;>
 
 # Docs
 
-Welcome to the documentation for the Urbit project, including the Azimuth identity layer, the Arvo operating system, and the Hoon programming language. This documentation is maintained by [Tlon](https://tlon.io) in a public [Github repository](https://github.com/urbit/docs). Issues and contributions are welcome.
+Welcome to the documentation for the Urbit project, including the Azimuth identity layer, the Arvo operating system, and the Hoon programming language. This documentation is maintained by [Tlon](https://tlon.io) in a public [GitHub repository](https://github.com/urbit/docs). Issues and contributions are welcome.
 
-Please make your way to the [Introduction](docs/introduction) for an overview of the docs.
+The documentation is organized into four high-level sections. Read on to get acquainted.
+
+## [Getting Started](/docs/getting-started)
+
+If you'd like to install Arvo, get an Azimuth point, and/or get on the Urbit network, you'll find guides here. Dive in, and don't be afraid to [ask for help](mailto:support@urbit.org)!
+
+The three pages below are meant to be followed in order.
+
+- The [Bridge section](/docs/getting-started/azimuth) is the first step. It tells you how to access your Azimuth identity using our client.
+- The [guide to installing Urbit](/docs/getting-started/installing-urbit) is the second step. It tells you how to get our software on your machine.
+- The [guide to booting a ship](/docs/getting-started/booting-a-ship) is the third and final step. It walks you through using what you did in the previous steps to get onto the Arvo network.
+
+In addition to those primary pages, the documents below are helpful but optional for using Urbit.
+
+- [Creating a Development Ship](/docs/getting-started/creating-a-development-ship), a walkthrough for creating an unnetworked a a platform for coding.
+- [Operating a Star](/docs/getting-started/creating-a-development-ship)
+
+## [Concepts](/docs/concepts)
+
+If you're looking for a high-level introduction to the Urbit project and its components, start here.
+
+- [Arvo vs. Azimuth](/docs/introduction/arvo-vs-azimuth), a quick explanation of the two parallel systems that compose Urbit.
+- [Technical Overview](/docs/introduction/technical-overview), a more in-depth description of the entire Urbit stack.
+- [Contributing](/docs/introduction/contributing), a guide to getting involved with the Urbit project.
+- [Community Tutorials](/docs/introduction/community-tutorials), a collection of unofficial but useful Urbit learning resources.
+- [Source Code](/docs/introduction/source-code-overview), an explanation of Urbit's constituent GitHub repos.
+- [Galaxies, Stars, and Planets](/docs/introduction/galaxies-stars-and-planets), an explanation of the Urbit address-space hierarchy.
```

Comment:
Azimuth* address-space hierarchy?

Reply:
That's not totally correct, because moons and comets have no Azimuth presence, and the article mentions them.